tests/integration/test_charm.py

```diff
@@ -64,18 +71,22 @@ async def test_build_and_deploy(ops_test: OpsTest):
         trust=True,
     )
 
+    await ops_test.model.integrate(OPENFGA, DB_APP)
+    await ops_test.model.wait_for_idle([OPENFGA, DB_APP], status="active", timeout=1000)
```

Comment:
Added this to ensure openfga gets active before relating with admin ui (see https://github.com/canonical/identity-platform-admin-ui-operator/actions/runs/8051508521/job/21991817378#step:4:514). Note that I didn't experience this issue on an ec2.

Reply:
Was this a one time error or did it happen consistently on the github runner? If it is the latter, then it looks like a bug and we should investigate it.

Reply:
It happened consistently on github runners, but I haven't experienced that on an ec2 instance, so it's difficult to reproduce.
I will run those tests multiple times on my setup to see if I can get into that.

Reply:
I wasn't able to reproduce it on an ec2, but a similar issue was observed on a terraform deployment: https://github.com/canonical/openfga-operator/issues/25
I updated it with my findings.